pkg/cli/cmd/group/groupswitch/switch.go

```diff
@@ -43,11 +43,11 @@ func NewCommand(factory framework.Factory) (*cobra.Command, framework.Runner) {
 		Short: "Switch default resource group scope",
 		Long: `Switch default resource group scope
 	
-	Radius workspaces contain a resource group scope, where Radius Applications and resources are deployed by default. The switch command changes the default scope of the workspace to the specified resource group name.
-	
-	Resource groups are used to organize and manage Radius resources. They often contain resources that share a common lifecycle or unit of deployment.
-			
-	Note that these resource groups are separate from the Azure cloud provider and Azure resource groups configured with the cloud provider.`,
+Radius workspaces contain a resource group scope, where Radius Applications and resources are deployed by default. The switch command changes the default scope of the workspace to the specified resource group name.
```

Comment:
The formatting for other commands is similar to the current version of this ([example](https://github.com/radius-project/radius/blob/a7e2e11ff08c650606a7b496f0a12903a182b878/pkg/cli/cmd/group/delete/delete.go#L47)). Is it broken for those as well?

Reply:
Thank you! I found a few more cases of broken formatting and pushed another commit.